client/my-sites/migrate/step-confirm-migration.jsx

```diff
@@ -112,6 +112,7 @@ class StepConfirmMigration extends Component {
 		const { targetSite, translate } = this.props;
 		const targetSiteDomain = get( targetSite, 'domain' );
 
+		// TODO: is the following "import everything" behaviour up to date?
```

Comment:
Is the following "import everything" behaviour up to date? I don't have enough context to understand if it can stay as-is, or if it can be deleted, or otherwise tweaked.